### DISCUSSION TITLE
[Feature Request] Settings Toggle for `Copied!` Notification

### DISCUSSION THREAD

**Discussion by FelipeRearden** [jglev/obsidian-carry-forward#12] [Ideas]
Hello @jglev !!!!

Would you mind creating a toggle in the setting to disable the notification `Copied!` while using one of the Carry-Forward commands ?

The notification is great ! This is something personal ....
- I have some concentration problems and the notification gives me some attention deficit since I use Carry-Forward many many times in my 5 - 8 hours daily in Obsidian

Thank you very much !!!!!!!!

**Comment by jglev** [comment]
Sure! I've addressed this in the upcoming [`v1.4.0`](https://github.com/jglev/obsidian-carry-forward/releases/tag/1.4.0-beta0), on your suggestion : )

**Reply by FelipeRearden** [reply]
> Sure! I've addressed this in the upcoming [`v1.4.0`](https://github.com/jglev/obsidian-carry-forward/releases/tag/1.4.0-beta0), on your suggestion : )

Thank you very much @jglev !!!!!!!!!
